### ISSUE TITLE
exif: createDate vs File Modification Date/Time

### ISSUE THREAD

**Issue by tjbaker** [gilesknap/gphotos-sync#150]
I suspect this is more of a Google Photos vs Plex issue, but here goes...

- I modified a pic via the Google Photos web to adjust the date.  Great, it now shows in the UI timeline correctly.
- I download the photo via gphotos-sync.  It sorted the photo according to the "File Modification Date/Time" exif tag, which has the user corrected date/time.  Still good.
- I add the photos to a Plex library, which uses the photo "Create Date" exif tag for sorting, so now it is back to square one.

Should Google Photos adjust the "Create Date" tag as well as the "File Modification Date/Time" tag?
Should gphotos-sync have an option to update the "Create Date" tag on local write?

**Comment by tjbaker** [comment]
Ugh, it looks like Plex has taken a pass over the photos, analyzed the metadata, and is now showing the dates as expected.

**Comment by ManuCart** [comment]
When you change the date on google photo. Google store this new date on a json file and google not modify the exif photo. So when you download the photo the exif is the same as the uploaded file.

**Comment by gilesknap** [comment]
I believe ManuCart is correct. 

At present, gphotos-sync does not even look in the EXIF (unless you use its comparison feature) let alone modify it. It only uses the metadata provided by the Google API. I'm sure Plex is just using the EXIF and that is your issue.

It would not be hard to write some Python to tidy up the EXIF in whatever fashion you see fit. In fact, I already wrote a very crude script a few years ago to do just that before I uploaded my historical library to Google. If you fancy having a go at this then https://github.com/gilesknap/ExifFolderTagger might give you a starting point.
